src/main/groovy/life/qbic/datamodel/dtos/business/facilities/Facility.groovy

```diff
@@ -0,0 +1,47 @@
+package life.qbic.datamodel.dtos.business.facilities
+
+/**
+ * <p>Controlled vocabulary for concise core facility labels.</p>
+ *
+ * <p>The following facilities are currently defined:</p>
+ *
+ * <ul>
+ *     <li>CFMB: Core Facility for Medical Bioanalytics</li>
+ *     <li>IMGAG: Institute for Medical Genetics and Applied Genomics</li>
+ *     <li>MGM: Institute for Medical Microbiology and Hygiene</li>
+ *     <li>QBIC: Quantitative Biology Center</li>
+ *     <li>PCT: Proteome Center Tübingen</li>
+ * </ul>
+ *
+ * @since 2.11.0
+ */
+enum Facility {
+
+    CFMB("Core Facility for Medical Bioanalytics"),
+    IMGAG("Institute for Medical Genetics and Applied Genomics"),
+    MGM("Institute for Medical Microbiology and Hygiene"),
+    QBIC("Quantitative Biology Center"),
+    PCT("Proteome Center Tübingen")
+
+    private final String value
```

Comment:
I think we could change that to something meaningful. This way we can extend the enum by other properties if we need them in the future. And it is clear what is meant here.
```suggestion
    private final String fullName
```